paddle/fluid/pybind/global_value_getter_setter.cc

```diff
@@ -144,44 +240,104 @@ void BindGlobalValueGetterSetter(pybind11::module *module) {
 
   py::class_<GlobalVarGetterSetterRegistry>(*module,
                                             "GlobalVarGetterSetterRegistry")
-      .def("__getitem__", &GlobalVarGetterSetterRegistry::Get)
-      .def("__setitem__", &GlobalVarGetterSetterRegistry::Set)
+      .def("__getpublic__", &GlobalVarGetterSetterRegistry::GetPublic)
+      .def("__getprivate__", &GlobalVarGetterSetterRegistry::GetPrivate)
+      .def("__setpublic__", &GlobalVarGetterSetterRegistry::SetPublic)
+      .def("__setprivate__", &GlobalVarGetterSetterRegistry::SetPrivate)
       .def("__contains__", &GlobalVarGetterSetterRegistry::HasGetterMethod)
       .def("keys", &GlobalVarGetterSetterRegistry::Keys)
-      .def("get", &GlobalVarGetterSetterRegistry::GetOrReturnDefaultValue,
+      .def("getpublic",
+           &GlobalVarGetterSetterRegistry::GetPublicOrReturnDefaultValue,
+           py::arg("key"), py::arg("default") = py::cast<py::none>(Py_None))
+      .def("getprivate",
+           &GlobalVarGetterSetterRegistry::GetPrivateOrReturnDefaultValue,
            py::arg("key"), py::arg("default") = py::cast<py::none>(Py_None));
 
   module->def("globals", &GlobalVarGetterSetterRegistry::Instance,
               py::return_value_policy::reference);
 }
 
-#define REGISTER_GLOBAL_VAR_GETTER_ONLY(var)                        \
+#define REGISTER_PRIVATE_GLOBAL_VAR_GETTER_ONLY(var, flag)          \
```

Comment:
In fact, your "var" is a flag and your 'flag' means 'is_public' ...

Rename 'flag's here around

Reply:
Done.